packages/package-json-lint/src/lint.js

```diff
@@ -1,7 +1,43 @@
-const { Logger } = require('@cerner/terra-cli');
+const fs = require('fs-extra');
+const rules = require('./rules');
+const PackageIssues = require('./issues/PackageIssues');
+const AggregateIssues = require('./issues/AggregateIssues');
+const { getRuleConfig, getConfig } = require('./config');
+const { getPathsForPackages } = require('./utilities');
 
-const logger = new Logger({ prefix: '[terra-functional-testing:wdio]' });
+const lint = ({ packageJsonData, config }) => {
+  const issues = [];
+  Object.entries(config.rules).forEach(([ruleId, ruleInformation]) => {
```

Comment:
could we just use filter here, instead of forEach + array.push?

Reply:
Hmmm. Would it need to be a combo of map and then filter?  Since we're not actually filtering Object.entries(config.rules) but instead transforming it (and then filtering it)

Reply:
ohh I see now, good point - we get slightly better readability with map + filter but it's two iterations - doing this in one iteration isn't that hard to read 🤷🏼 could go either way